k8s/prod/common/em/em-anno.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: em-anno
+  namespace: em
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.em-anno: glob:prod-*
+spec:
+  releaseName: em-anno
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: em-anno
+    version: 0.1.11
+  values:
+    anno:
+      replicas: 2
+      applicationInsightsInstrumentKey: "039faf8c-4682-432e-8fc5-f9735bb4b962"
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/em/anno:prod-ad59cc6e
+      ingressHost: em-anno-prod.service.core-compute-prod.internal
```

Comment:
You still need to override https://github.com/hmcts/rpa-em-annotation-api/blob/master/charts/em-anno/values.yaml#L19
as IDAM Prod url is https://idam-api.platform.hmcts.net (without prod)